tests/unit/field/test_text_field.py

```diff
@@ -242,6 +242,20 @@ def test_load_embeddings():
     assert torch.all(torch.eq(field.embedding_matrix[1:3], true_embeddings))
 
 
+def test_load_embeddings_with_extra_tokens():
+    field = TextField.from_embeddings(
+        embeddings="tests/data/dummy_embeddings/test.txt",
+        pad_token=None,
+        unk_init_all=False,
+        additional_vocab_tokens=['<a>', '<b>', '<c>']
+    )
+    dummy = "a test ! <a> <b> "
+    field.setup([dummy])
+    assert '<a>' in field.vocab and '<b>' in field.vocab
+    assert '<c>' not in field.vocab
+    assert field.embedding_matrix[field.vocab['<a>']].size(-1) == 4
```

Comment:
assert than `<b>` is also part of the embedding_matrix?

Reply:
done